src/uri/mod.rs

```diff
@@ -673,6 +673,22 @@ impl Uri {
     fn has_path(&self) -> bool {
         !self.path_and_query.data.is_empty() || !self.scheme.inner.is_none()
     }
+
+    /// make a Uri::Builder from the Uri to modify part of it
+    pub fn to_builder(self) -> Builder {
```

Comment:
naming nitpick, this would conventionally be `into_builder`

Reply:
Thanks. It makes sense. Fixed at 1db81950d55a642dabd937fe2acf012e22eebfd8